### PR TITLE
KAA-1160 Glossary update started.

### PR DESCRIPTION
**Pull request by hunch7**
KAA-1160 Glossary update process started.

### CODE REVIEW

**Comment by kaa-leeroy** [comment]
Can one of the admins verify this patch?

File: doc/Glossary/index.md [hunk 1]
```diff
@@ -5,74 +5,236 @@ permalink: /:path/
...
-| U                                                       |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
-| User verifier                                           | A server component that handles the user verification. Refer to User verifiers for more information.
```
Comment (akokhanovskyi) [inline doc/Glossary/index.md:72]: A virtual representation of an Endpoint, that is part of the [actor gateway]({{root_url}}Glossary/#actor-gateway) architecture.
An actor typically communicates with the actual device and represents it to [Kaa server]({{root_url}}Glossary/#kaa-server).

Comment (akokhanovskyi) [inline doc/Glossary/index.md:72]: "Actor gateway" is in fact a component, not a type of deployment. And you should explain what it is by telling what it does, not when it's used. You can mention that is is especially useful under these and that circumstances in 2-nd or 3-rd phrase.

**Comment by Oldnickus** [comment]
@kaa-leeroy add to whitelist

File: gh-pages-stub/scripts/create_global_toc.rb [hunk 2]
```diff
@@ -85,7 +85,8 @@ def sortSubitems(node)
   def loadDoc(key)
     Dir.glob("#{key}/**/index.md") do |md_file|
       dirname = File.dirname(md_file)
-      header = YAML.load(loadHeader(md_file))
+      puts md_file
+	header = YAML.load(loadHeader(md_file))
```
Comment (rasendubi) [inline gh-pages-stub/scripts/create_global_toc.rb]: I believe this file shouldn't be changed in the PR

Comment (hunch7) [reply gh-pages-stub/scripts/create_global_toc.rb]: Fixed.

**Comment by Oldnickus** [comment]
@kaa-leeroy test this please

**Comment by vadimol** [comment]
@kaa-leeroy test this please

**Comment by hunch7** [comment]
Replaced by https://github.com/kaaproject/kaa/pull/1171

**Pull request closed without merging**